tests/unit_tests/detection_utils_test.py

```diff
@@ -24,7 +24,7 @@ def setUp(self):
     def test_visualization(self):
 
         valid_loader = coco2017_val(dataloader_params={"batch_size": 16})
-        trainer = Trainer("visualization_test", device=self.device)
+        trainer = Trainer("visualization_test")
```

Comment:
how is that related?

Reply:
Not related, but I ran all of our unit tests (including those outside the suite) and fixed anything that was crashing.
Passing a divece through trainer was deprecated and causes to crash since few versions ago.